awscognito/cognito_users.py

```diff
@@ -0,0 +1,76 @@
+# pylint: disable=global-statement,redefined-outer-name
+""" Script used to create|delete AWS Cognito user """
+import argparse
+import sys
+
+
+def create_user(client, profile, user):
+    """ Creates a new user in the specified user pool """
+    response = client.admin_create_user(
+        UserPoolId=profile.pool_id,
+        Username=user.email,
+        UserAttributes=[
+            {"Name": "email", "Value": user.email},
+            {"Name": "custom:name", "Value": user.name},
+        ],
+        ValidationData=[{"Name": "string", "Value": "string"},],
+        MessageAction="RESEND",
```

Comment:
What does "RESEND" do? (Sorry I didn't check the document.)

Reply:
I think we can ignore this if we will set "email_verified" later.